### PR TITLE
Content Edits in About Us

### PR DESCRIPTION
**Pull request by garimavs**
## Fixes

Documentation edits on the About Us page.

## Types of changes

- [ ] Bug/Hot fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [x] Refactoring Old Code ( non-breaking change replacing already existing code )
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have read the **CONTRIBUTING** guidelines in repository.
- [ ] My pull request has a descriptive title and is attached to an existing issue.
- [x] I haven't repeated the code and it follows established code style of repository.
- [x] My branch is up-to-date with upstream branch.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by king-11** [comment]
Sorry for the delay taking a look now

File: components/about-us/aboutuscard.vue [hunk 1]
```diff
@@ -88,14 +89,14 @@
...
-          sagittis id augue. Sed molestie hendrerit ex eu blandit. Nam ut
-          imperdiet purus. Fusce pretium sollicitudin dapibus. Etiam gravida
-          fermentum orci semper luctus. Maecenas est arcu, sagittis non purus
-          vitae, blandit egestas nibh.
+          Whether Mobile Development, Desktop Development, Web Development, Game
+          Development, or Cloud Development, we manufacture all units here!
+          Along the lines of development, we enjoy discussions and
+          collaborations too. Regular DevTalks for discussions and doubts about
+          new tech and associating over Open Source provides us with great
+          opportunities of expanding our colossal network. DevOps with GitLab,
```
Comment (king-11) [inline components/about-us/aboutuscard.vue:97]: Let's change devops with gitlab to multiverse of languages

File: components/about-us/aboutuscard.vue [hunk 2]
```diff
@@ -125,14 +126,15 @@
...
-          vitae, blandit egestas nibh.
+          Our motto is constantly learning and revamping. Hackalog, our very own
+          portal for maintaining logs of our Hackathon, helps us archive
+          hackathon details and leaderboards. Lite Hai, our institute's app, is
+          every student's guide to various student bodies, fests, workshops; and
+          it also comes with Google Maps for easy movement within the campus.
+          Our official website for the Software Development Group is a
+          storehouse for various software development blogs and our achievements
+          and projects. Our insatiable appetite towards developing adores
+          creating Telegram Bots would be fair to say.
```
Comment (king-11) [inline components/about-us/aboutuscard.vue:137]: Surely they are cool :)

**Review by king-11 — CHANGES_REQUESTED** [review]

File: components/about-us/aboutuscard.vue (hunk 1 above)
Comment (garimavs) [reply components/about-us/aboutuscard.vue:97]: Okay, will update it!!

**Review by king-11 — APPROVED** [review]

**Pull request merged**